acoustid/data/track.py

```
@@ -14,18 +14,17 @@
 logger = logging.getLogger(__name__)
 
 
-def resolve_track_gid(conn, gid):
-    # type: (FingerprintDB, str) -> Optional[int]
-    query = sql.select(
-        [schema.track.c.id, schema.track.c.new_id], schema.track.c.gid == gid
+def resolve_track_gid(conn: FingerprintDB, gid: str) -> int | None:
+    query = sql.select(schema.track.c.id, schema.track.c.new_id).where(
+        schema.track.c.gid == gid
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)